src/components/earn/PoolCardTri/index.tsx

```diff
@@ -160,22 +140,59 @@ const DefaultPoolCardtri = ({
           <StyledMutedSubHeader justifyContent="flex-start">APR</StyledMutedSubHeader>
           <PoolCardTriRewardText apr={apr} inStaging={inStaging} nonTriAPRs={nonTriAPRs} isLegacy={isLegacy} />
         </AprContainer>
-        <RowActionsContainer>{renderActionsContainer()}</RowActionsContainer>
+        <RewardsContainer>
+          <StyledLongClaimableHeader>Claimable rewards</StyledLongClaimableHeader>
+          <StyledShortClaimableHeader>Rewards</StyledShortClaimableHeader>
+
+          <StyledRewardsAmountContainer>
+            {enableClaimButton ? (
+              <>
+                {!noTriRewards && (
+                  <RewardColumn>
+                    <StyledCurrencyLogo currency={TRI[ChainId.AURORA]} size="14px" />
+                    <StyledRewardAmount>
+                      <CountUp
+                        enabled={earnedAmount?.greaterThan(BIG_INT_ZERO) ?? false}
+                        value={parseFloat(earnedAmount?.toFixed(6) ?? '0')}
+                        decimalPlaces={2}
+                      />
+                    </StyledRewardAmount>
+                  </RewardColumn>
+                )}
+                {earnedNonTriRewards?.map(({ amount, token }) => (
+                  <RewardColumn key={token.address}>
+                    <StyledCurrencyLogo currency={token} size="14px" />
+                    <StyledRewardAmount>
+                      <CountUp
+                        enabled={amount?.greaterThan(BIG_INT_ZERO) ?? false}
+                        value={parseFloat(amount?.toFixed(6) ?? '0')}
+                        decimalPlaces={2}
+                      />
+                    </StyledRewardAmount>
+                  </RewardColumn>
+                ))}
+              </>
+            ) : (
+              <TYPE.body>-</TYPE.body>
+            )}
+          </StyledRewardsAmountContainer>
+        </RewardsContainer>
+        <ButtonGold
+          padding="8px"
+          borderRadius="8px"
+          maxWidth={isMobileOnly ? '55px' : '74px'}
+          height="30px"
+          onClick={event => handleClaimClick(event)}
+          disabled={!enableClaimButton}
+          justifySelf="start"
+          fontSize={isMobileOnly ? '14px' : '16px'}
+        >
+          Claim
+        </ButtonGold>
         <DetailsContainer>{showMore ? <ChevronUp size="15" /> : <ChevronDown size="15" />}</DetailsContainer>
       </CardContainer>
 
-      {showMore && (
-        <div>
-          <ExpandableStakedContainer>
-            <Text>{t('earn.totalStaked')}</Text>
-            <TYPE.white fontWeight={500}>{`$${totalStakedInUSDFriendly}`}</TYPE.white>
-          </ExpandableStakedContainer>
-          <ExpandableActionsContainer>
-            <Text>Manage this Farm</Text>
-            {renderActionsContainer()}
-          </ExpandableActionsContainer>
-        </div>
-      )}
+      {showMore && <div></div>}
```

Comment:
What's this?

Reply:
Was intended to show that the expandable section is indeed working, but removed now, not neccesary